### PR TITLE
fix: voicea bugs

### PR DESCRIPTION
**Pull request by srinijay98**
# COMPLETES #< INSERT LINK TO ISSUE >

## This pull request addresses

< DESCRIBE THE CONTEXT OF THE ISSUE >
Fix voicea bugs and refactor plugin

## by making the following changes

< DESCRIBE YOUR CHANGES >
1) Inject webex object properly
2) fix response body object destructuring
3) extend voicea with Webexplugin instead of LLM plugin

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

< ENUMERATE TESTS PERFORMED, WHETHER MANUAL OR AUTOMATED >

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by arun3528 — DISMISSED** [review]
We should move away from extending llm on voicia and directly using the plugin llm there similar to meetings

File: packages/node_modules/@webex/internal-plugin-llm/src/llm.ts [hunk 1]
```diff
@@ -83,7 +81,10 @@ export default class LLMChannel extends (Mercury as any) implements ILLMChannel
   }).then((res: { body: { webSocketUrl: string; binding: string; }; }) => {
     this.webSocketUrl = res.body.webSocketUrl;
     this.binding = res.body.binding;
-  });
+  }).catch((e:any) => {
+    console.error('Error connecting to websocket');
+    console.error(e);
```
Comment (mkesavan13) [inline packages/node_modules/@webex/internal-plugin-llm/src/llm.ts:86]: Shouldn't we be using Logger plugin instead of `console.error` ?

**Review by mkesavan13 — CHANGES_REQUESTED** [review]
Looks good to me. Just couple of changes,

- We need not checkin webex.min.js. Kindly remove it from the commit. 
- If all these changes relate to internal-plugin-voicea, please squash all commits and mention the plugin in the commit like `fix(interna-plugin-voicea): COMMIT_MESSAGE` as stated in the [contribution guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md) and that helps in having a cleaner history and change log

File: packages/node_modules/@webex/internal-plugin-voicea/src/voicea.ts [hunk 2]
```diff
@@ -87,9 +95,10 @@ export class VoiceaChannel extends LLMChannel implements IVoiceaChannel {
           translations: voiceaPayload.translations,
           transcript: {
             csis: voiceaPayload.csis,
-            text: voiceaPayload.data
+            text: voiceaPayload.transcript.text,
+            transcriptLanguageCode: voiceaPayload.transcript.transcript_language_code
           },
-          timestamp: millisToMinutesAndSeconds(voiceaPayload.transcript.end_millis - voiceaPayload.transcript.start_millis)
+          timestamp: millisToMinutesAndSeconds(voiceaPayload.transcript.end_millis)
```
[the remark this answers is not in the document]
Comment (srinijay98) [reply packages/node_modules/@webex/internal-plugin-voicea/src/voicea.ts:101]: my calculation was wrong
transcription starts at 10000ms and ends at 15000 -> this means the transcript timestamp is 150000
instead 15000-10000 = 5000ms = 5s was a wrong timestamp

**Review by Coread — APPROVED** [review]

**Review by mkesavan13 — APPROVED** [review]

**Pull request merged**